script.js

```diff
@@ -219,15 +219,15 @@ function formValidate(event) {
     event.preventDefault();
   } else if (email.value !== email.value.toLowerCase()) {
     errorMessage.style.display = 'block';
-    errorMessage.innerHTML = 'Please write your valid email address in lowercase.';
+    errorMessage.innerHTML = 'Please write your valid Email address in lowercase.';
     event.preventDefault();
   } else if (/[A-Z]/.test(email.value)) {
     errorMessage.style.display = 'block';
-    errorMessage.innerHTML = 'Please use Lowercase in your email address';
+    errorMessage.innerHTML = 'Please use Lowercase in your Email address';
     event.preventDefault();
   } else if (message.value === '') {
     errorMessage.style.display = 'block';
-    errorMessage.innerHTML = 'Please write your message.';
+    errorMessage.innerHTML = 'Please write your Message.';
```

Comment:
_ [ ] Nice work so far 👍🏿  on your project, but I notice that when I submit the validation does not work even do I add an uppercase email address, I kindly suggest you validation to check for any uppercase in the email input and throw error